### PR TITLE
Incremental CS checks in 2.x branches

### PR DESCRIPTION
**Pull request by Majkl578**
Enahances and replaces #7350.

### CODE REVIEW

**Comment by alcaeus** [comment]
Can you please add information about this to the contribution docs? We should tell contributors how they can check their changes only without running phpcs on the entire code.

**Review by ostrolucky — DISMISSED** [review]

**Comment by ostrolucky** [comment]
That can be done in separate PR. Let's merge this as it is so this PR doesn't sit around even longer

File: lib/Doctrine/ORM/EntityManager.php [hunk 1]
```diff
@@ -148,9 +148,9 @@
      */
     protected function __construct(Connection $conn, Configuration $config, EventManager $eventManager)
     {
-        $this->conn              = $conn;
+        $this->conn          = $conn;
         $this->config            = $config;
-        $this->eventManager      = $eventManager;
+        $this->eventManager    = $eventManager;
```
Comment (greg0ire) [inline lib/Doctrine/ORM/EntityManager.php]: This looked funny, and looks even funnier now

Comment (ostrolucky) [reply lib/Doctrine/ORM/EntityManager.php]: indeed indentation is broken here, not sure why this change

Comment (ostrolucky) [reply lib/Doctrine/ORM/EntityManager.php]: seems this is just for test

Comment (greg0ire) [reply lib/Doctrine/ORM/EntityManager.php]: Oh ok

**Review by alcaeus — DISMISSED** [review]
Agree with @ostrolucky. This is already better than before, we can update documentation later.

**Review by Ocramius — DISMISSED** [review]
:ship:

**Comment by Ocramius** [comment]
Ah, there are still some failures left, sorry:

```
FILE: lib/Doctrine/ORM/EntityManager.php
----------------------------------------------------------------------
FOUND 3 ERRORS AFFECTING 3 LINES
----------------------------------------------------------------------
 151 | ERROR | [x] Equals sign not aligned with surrounding
     |       |     assignments; expected 9 spaces but found 10
     |       |     spaces
     |       |     (Generic.Formatting.MultipleStatementAlignment.NotSame)
 153 | ERROR | [x] Equals sign not aligned with surrounding
     |       |     assignments; expected 1 space but found 4
     |       |     spaces
     |       |     (Generic.Formatting.MultipleStatementAlignment.NotSame)
 293 | ERROR | [x] There must be a single space after a NOT operator;
     |       |     0 found
     |       |     (Generic.Formatting.SpaceAfterNot.Incorrect)
----------------------------------------------------------------------
PHPCBF CAN FIX THE 3 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------
```

**Comment by ostrolucky** [comment]
Yeah just removal of last commit needs to be done. Can be done by merger?

**Comment by lcobucci** [comment]
@Majkl578 @Ocramius I've rebased it but have a meeting now and can't wait for the build to merge it, please do it!

**Review by alcaeus — APPROVED** [review]

**Pull request merged**